src/main/java/core/basesyntax/service/impl/MyFileReaderImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyFileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(filePath);
+        if (inputStream == null) {
+            throw new RuntimeException("File not found: " + filePath);
+        }
+        try (BufferedReader br = new BufferedReader(new InputStreamReader(inputStream))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
```

Comment:
lets find an easier way to read form file

Reply:
check this point

Reply:
I tried to do it in other ways, but there is an error with file shielding, I don't understand why

Reply:
I know how to work with files, but in this particular case, I couldn't read the file, so I had to use AI to help me.